Api/Data/CompanySearchResultInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Devall\Company\Api\Data;
+
+use Devall\Company\Api\Data\CompanyInterface;
+use Magento\Framework\Api\SearchResultsInterface;
+
+interface CompanySearchResultInterface extends SearchResultsInterface
+{
+    /**
+     * @return \Devall\Company\Api\Data\CompanyInterface[]
+     */
+    public function getItems();
```

Comment:
Missing return type declaration

Reply:
it gives me an error when doing setup:di:compile

Fatal error: Declaration of Magento\Framework\Api\SearchResults::getItems() must be compatible with Devall\Company\Api\Data\CompanySearchResultInterface::getItems():